source/server/admin/admin.h

```diff
@@ -128,6 +129,14 @@ class AdminImpl : public Admin,
     return request_id_extension_;
   }
   const AccessLog::InstanceSharedPtrVector& accessLogs() override { return access_logs_; }
+  bool acceptTargetRoute(absl::string_view route_name) const {
+    bool is_route_allowed = std::any_of(allow_listed_routes_.begin(), allow_listed_routes_.end(),
+                                        [&route_name](const Matchers::StringMatcherPtr& matcher) {
+                                          return matcher->match(route_name);
```

Comment:
not sure but do we need to take a ref of string_view? (not verified by myself locally. just my guess)
```suggestion
                                        [route_name](const Matchers::StringMatcherPtr& matcher) {
                                          return matcher->match(route_name);
```